### DISCUSSION TITLE
Can I add an unofficial version to a port?

### DISCUSSION THREAD

**Discussion by serg06** [microsoft/vcpkg#18722] [Q&A] [answered]
The [RxCpp repo](https://github.com/ReactiveX/RxCpp) is no longer maintained, but it has had many useful changes since the last release like [fwd headers](https://github.com/ReactiveX/RxCpp/commit/6e2eee6b68ae14297ca4c178a33aa529a6ab3997#diff-9e468e902c8c7359df073c50d6735f31c34a0b4d719d0c81d0ab35f25b489913). 

Can I make a port of the latest commit (not an official release) and update the version from "4.1.0-1" to something like "4.1.0-2"?

**Comment by mathisloge** [comment]
I think the current policy is not to add unmaintained libraries and add them instead into custom registries. however it doesn't look so unmaintained. The last commits are from 2020. Only the last release is 3 years ago. 

In my opinion it would be better to try to trigger a release upstream to include all the changes since to 2018.

**Reply by vicroms** [reply]
If the upstream port is no longer doing tagged versions it may make sense to switch to rolling releases

I would try @mathisloge's suggestion of requesting a new release upstream and if that doesn't work then update the port using `version-date` instead.

**Comment by serg06** [comment, marked as answer]
https://github.com/microsoft/vcpkg/discussions/18722#discussioncomment-1048478

**Comment by johnhere6754** [comment]
4chan X [control biometrico](https://www.genera.cl/conoce-como-funciona-el-control-de-asistencia-biometrico/sistemas-de-control-de-asistencia/)- Unofficial Port version history - 2 versions. Be careful with old versions! You should always use the latest version of an add-on.
